schemas/events/v1/events.json

```diff
@@ -55,6 +55,22 @@
       "type": "string",
       "pattern": "https://console.redhat.com/api/schemas/.*"
     },
+    "title": {
+      "description": "Human readable text to identify the event",
+      "type": "string",
+      "examples": [
+        "5 Recomendations for system MySystem",
+        "12 Policies triggered for system MySystem",
+      ]
+    },
+    "link": {
```

Comment:
Should we call that 'href'? Also make the description potentially more clear that this href is again for the entire message and not (necessarily) to individual events in the payload.
Would such a href even work in the sense that if the payload is a policies and an advisor event it would need to point to two different paths at once?

Reply:
IMO it does not fit the `dataref extension` - this is not an url with the contents of the data.
It's meant to an url to human readable content (i.e. https://console.redhat.com/insights/inventory/06b77ad5-01cb-4e9f-acd1-5ede004a432b)

`html_ref`  or `htmlref` sounds fine to me,